PivotGridDataSource: Fix calculateSummaryValue Code & Add Example Link (#8792)
DevExpress/devextreme-documentation: api-reference/30 Data Layer/PivotGridDataSource/1 Configuration/fields/calculateSummaryValue.md

---

> You can configure **calculateSummaryValue** to apply custom aggregation logic to Total and Grand Total rows/columns. For more information, refer to the following example:

Reviewer: ```suggestion
You can configure **calculateSummaryValue** to apply custom aggregation logic to total and grand total rows and columns. For an example, see the following topic:
```

Author: The link below this line is a GitHub example, not a topic. Maybe we can slightly shorten the last sentence as follows?
```suggestion
You can configure **calculateSummaryValue** to apply custom aggregation logic to total and grand total rows and columns. For more information, see the following example:
```
